docs/using-nativeaot/compiling.md

```diff
@@ -114,42 +115,42 @@ Functions in other WebAssembly modules can be imported and invoked using `DllImp
 [DllImport("*")]
 static extern int random_get(byte* buf, uint size);
 ```
-Be default emscripten will create a WebAssembly import for this function, importing from the `env` module.  This can be controlled with `WasmImport` items in the project file.  For example
+By default emscripten will create a WebAssembly import for this function, importing from the `env` module.  This can be controlled with `WasmImport` items in the project file. For example
```

Comment:
Showing my age with double space after .s :-)
```suggestion
By default emscripten will create a WebAssembly import for this function, importing from the `env` module. This can be controlled with `WasmImport` items in the project file. For example
```